pizza_cutter_metadetect/masks.py

```diff
@@ -0,0 +1,337 @@
+    udecmin, udecmax, uramin, uramax : float
+        The min and max values for ra and dec indicating the unique coadd tile
+        region.
+
+    Returns
+    -------
+    flag : bool array
+        The boolean flag array with True indicating the (ra,dec) is in the
+        unique coadd tile region.
+    """
+    ra = ra.copy()
+
+    msk = (ra < 0) & np.isfinite(ra)
+    while np.any(msk):
+        ra[msk] = ra[msk] + 360.0
+        msk = (ra < 0) & np.isfinite(ra)
+
+    msk = (ra > 360) & np.isfinite(ra)
+    while np.any(msk):
+        ra[msk] = ra[msk] - 360.0
+        msk = (ra > 360) & np.isfinite(ra)
+
+    if crossra0 == 'Y':
+        uramin = uramin - 360.0
+        msk = ra > 180.0
+        ra[msk] -= 360
+
+    in_coadd = (
+        (ra > uramin)
+        & (ra <= uramax)
+        & (dec > udecmin)
+        & (dec <= udecmax)
+    )
+    msk = np.isfinite(ra) & np.isfinite(dec)
+    in_coadd[~msk] = False
+
+    return in_coadd
+
+
+def get_slice_bounds(
+    *, orig_start_col, orig_start_row, central_size, buffer_size, coadd_dims,
+):
+    """Get the unique pixel bounds for each slice.
+
+    Typically, this will simply cut the buffers. However, on the edge of a tile,
+    the buffer closest to the edge will be kept.
+
+    Note that all coordinates are 0-indexed pixel coordinates.
+
+    Parameters
+    ----------
+    orig_start_col : int
+        The starting col for the slice in the coadd coordinates.
+    orig_start_row : int
+        The starting row for the slice in the coadd coordinates.
+    central_size : int
+        The size of the central region of the slice.
+    buffer_size : int
+        The size of the buffer around each slice.
+    coadd_dims : 2-tuple of ints
+        The dimension of the coadd image from which the slice was made.
+
+    Returns
+    -------
+    slice_bnds : dict
+        A directory with keys 'min_row', 'max_row', 'min_col', and 'max_col'.
+    """
+    slice_dim = central_size + 2*buffer_size
+    if orig_start_row == 0:
+        min_slice_row = 0
+        max_slice_row = slice_dim - buffer_size
+    elif orig_start_row + slice_dim == coadd_dims[0]:
+        min_slice_row = buffer_size
+        max_slice_row = slice_dim
+    else:
+        min_slice_row = buffer_size
+        max_slice_row = slice_dim - buffer_size
+
+    if orig_start_col == 0:
+        min_slice_col = 0
+        max_slice_col = slice_dim - buffer_size
+    elif orig_start_col + slice_dim == coadd_dims[1]:
+        min_slice_col = buffer_size
+        max_slice_col = slice_dim
+    else:
+        min_slice_col = buffer_size
+        max_slice_col = slice_dim - buffer_size
+
+    return {
+        "min_row": min_slice_row,
+        "max_row": max_slice_row,
+        "min_col": min_slice_col,
+        "max_col": max_slice_col,
+    }
+
+
+def _convert_ra_dec_vals_to_healsparse(ra, dec, vals, healpix_nside):
+    hs_msk = healsparse.HealSparseMap.make_empty(
+        32, healpix_nside, np.int32, sentinel=0
```

Comment:
@erykoff is this sentinel right? 

I am confused as to if the sentinel should represent a pixel with no flags, a missing pixel, or something else. 

These files are meant to be combined into one giant mask where zero is observed and non-zero is masked.

Reply:
Or maybe what is the fill value for logical or? 0?